src/cli/cli.cpp

```diff
@@ -7618,6 +7677,16 @@ template <> otError Interpreter::Process<Cmd("unsecureport")>(Arg aArgs[])
             error = ProcessSet(aArgs + 1, otIp6RemoveUnsecurePort);
         }
     }
+    /**
+     * @cli unsecure get
+     * @code
+     * unsecure get
+     * 1234
+     * Done
+     * @endcode
+     * @par api_copy
```

Comment:
Let's not use api_copy here.  The API says it returns a pointer while this returns the port number(s), so let's use a custom description instead